tests/conftest.py

```diff
@@ -8,8 +9,12 @@
 
 from hydromt_wflow import WflowModel, WflowSedimentModel
 
+SUBDIR = ""
+if platform.system().lower() != "windows":
+    SUBDIR = "linux64"
+
```

Comment:
There does not seem to be a windows folder equivalent, so I wonder if this is necessary? I understand that might be out of scope for this PR but in general it is best to try and make things cross platform so these things are not necessary. I'm happy to help you with this if you want at some point.